### PR TITLE
multi_nics_verify: fix cannot get enough ip of nics for windows guest

### PR DESCRIPTION
**Pull request by kimi1978wy**
Increase the timeout for getting ip and boot time.

Signed-off-by: Yu Wang <wyu@redhat.com>

id:1709328

### CODE REVIEW

**Comment by kimi1978wy** [comment]
(01/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.multi_nics_verify.q35: PASS (349.54 s)
 (02/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (1832.61 s)
 (03/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_nics_verify.q35: PASS (323.79 s)
 (04/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (2276.46 s)
 (05/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_nics_verify.q35: PASS (310.03 s)
 (06/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (2001.94 s)
 (07/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.multi_nics_verify.q35: PASS (254.10 s)
 (08/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.x86_64.sp1.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (317.24 s)
 (09/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_nics_verify.q35: PASS (470.14 s)
 (10/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (519.67 s)
 (11/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.multi_nics_verify.q35: PASS (224.52 s)
 (12/12) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.1.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (486.79 s)

**Comment by kimi1978wy** [comment]
(1/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1236.42 s)
 (2/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1410.65 s)
 (3/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.q35: PASS (1395.94 s)
 (4/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.multi_nics_verify.q35: PASS (297.34 s)
 (5/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win7.i386.sp1.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (65.43 s)
 (6/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.multi_nics_verify.q35: PASS (249.68 s)
 (7/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (151.83 s)
 (8/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.multi_nics_verify.q35: PASS (231.89 s)
 (9/9) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.i386.1.io-github-autotest-qemu.multi_nics_verify.with_multiqueue.q35: PASS (131.92 s)

**Comment by vivianQizhu** [comment]
@kimi1978wy Please fix conflict.

**Comment by kimi1978wy** [comment]
@vivianQizhu conflict solved , and re-test, all pass

 (05/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_nics_verify: PASS (145.47 s)
 (06/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_nics_verify.with_multiqueue: PASS (390.17 s)
 (07/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.multi_nics_verify: PASS (259.09 s)
 (08/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.i386.io-github-autotest-qemu.multi_nics_verify.with_multiqueue: PASS (149.30 s)
 (09/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_nics_verify: PASS (313.04 s)
 (10/10) Host_RHEL.m8.u1.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_nics_verify.with_multiqueue: PASS (598.75 s)

File: qemu/tests/cfg/multi_nics_verify.cfg [hunk 1]
```diff
@@ -4,10 +4,14 @@
...
     variants:
         - @default:
         - with_multiqueue:
             queues = ${smp}
             vt_ulimit_nofile = 10240
-            Windows..i386:
-                nics_num = 8
+            Windows:
+                smp = 4
+                queues = 4
```
Comment (vivianQizhu) [inline qemu/tests/cfg/multi_nics_verify.cfg]: @kimi1978wy Would you help explain why set fixed smp and queues for Windows?

Comment (kimi1978wy) [reply qemu/tests/cfg/multi_nics_verify.cfg]: It is because of windows memory limited

**Pull request merged**